ui/src/App.tsx

```diff
@@ -5,6 +5,9 @@ import { createHashRouter, RouterProvider } from 'react-router-dom';
 import Pipeline from './app/pipeline';
 import { Helmet } from 'react-helmet';
 import { useGetSystemQuery } from './services/api';
+import { Toaster } from '@/components/ui/toaster';
+import { useToast } from '@/hooks/use-toast';
+import { useEffect } from 'react';
```

Comment:
```suggestion
import { toast } from 'sonner';
```

we already have the [sonner](https://ui.shadcn.com/docs/components/sonner) component in use elsewhere in the app, so could we continue to use that instead? 

this would mean we don't need the `toast/toaster` components or the `use-toast` hook either.